wallet/tests/wallet.rs

```diff
@@ -36,14 +36,15 @@ fn parse_descriptor(s: &str) -> (Descriptor<DescriptorPublicKey>, KeyMap) {
         .expect("failed to parse descriptor")
 }
 
-// The satisfaction size of a P2WPKH is 112 WU =
-// 1 (elements in witness) + 1 (OP_PUSH) + 33 (pk) + 1 (OP_PUSH) + 72 (signature + sighash) + 1*4 (script len)
-// On the witness itself, we have to push once for the pk (33WU) and once for signature + sighash (72WU), for
-// a total of 105 WU.
-// Here, we push just once for simplicity, so we have to add an extra byte for the missing
-// OP_PUSH.
+// The satisfaction size of a P2WPKH is 108 WU =
+// 1 (elements in witness) + 1 (OP_PUSH) + 33 (pubkey) + 1 (OP_PUSH) + 72 (signature + sighash).
+// TODO: tests won't pass with 108 but will with 106.
 const P2WPKH_FAKE_WITNESS_SIZE: usize = 106;
 
+/// The satisfaction size of a P2PKH is 107 WU =
+/// 1 (OP_PUSH) + 72 (signature + sighash) + 1 (OP_PUSH) + 33 (pubkey).
```

Comment:
Not technically weight units

```suggestion
/// The satisfaction size of a P2PKH is 107 =
/// 1 (OP_PUSH) + 72 (signature + sighash) + 1 (OP_PUSH) + 33 (pubkey).
```